crates/factor-outbound-networking/src/runtime_config/spin.rs

```diff
@@ -41,21 +45,62 @@ impl SpinTlsRuntimeConfig {
         &self,
         table: &impl GetTomlValue,
     ) -> anyhow::Result<Option<super::RuntimeConfig>> {
-        let Some(tls_configs) = self.tls_configs_from_table(table)? else {
+        let maybe_blocked_networks = self
+            .blocked_networks_from_table(table)
+            .context("parsing blocked_networks")?;
```

Comment:
Typically the error message would be something like:

```suggestion
            .context("failed to parse `blocked_networks` table")?;
```

The terse "parsing blocked_networks" will be hard to understand outside the scope of this code.

Most of the errors in this file could use a bit more context to help users understand what's happening.